dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md

```diff
@@ -0,0 +1,163 @@
+---
+title: "Create a new app with Visual Studio tutorial"
+description: Follow this tutorial to learn how to create a new Windows Forms app for .NET with Visual Studio 2019.
+ms.date: 10/26/2020
+ms.topic: tutorial
+dev_langs: 
+  - "csharp"
+  - "vb"
+---
+
+# Tutorial: Create a new WinForms app (Windows Forms .NET)
+
+In this short tutorial, you'll learn how to create a new Windows Forms (WinForms) app with Visual Studio. Once the initial app has been generated, you'll learn how to add controls and how to handle events. By the end of this tutorial you'll have a simple app that adds names to a list box.
+
+[!INCLUDE [desktop guide under construction](../../includes/desktop-guide-preview-note.md)]
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> - Create a new WinForms app
+> - Add controls to a form
+> - Handle control events to provide app functionality
+> - Run the app
+
+## Prerequisites
+
+- [Visual Studio 2019 16.8 Preview](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+winforms)
```

Comment:
```suggestion
- [Visual Studio 2019 version 16.8 Preview](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2019+desktopguide+winforms)
```

This will need to be updated as soon as GA is released. Then I'd say "...version 16.8 or later".